test/verify/machineslib.py

```diff
@@ -1282,11 +1291,15 @@ def fill(self):
 
             if self.is_filesystem_location:
                 expected_source_type = 'Local Install Media'
+            elif self.is_disk_image:
+                expected_source_type = 'Existing Disk Image'
             else:
                 expected_source_type = 'URL'
             b.select_from_dropdown("#source-type", expected_source_type)
             if self.is_filesystem_location:
                 b.set_file_autocomplete_val("#source-file", self.location)
+            elif self.is_disk_image:
+                b.set_file_autocomplete_val("#source-disk", self.location)
```

Comment:
To avoid an ever-growing `self.is_*` collection, you could just have a python dict somewhere that maps e. g. `{ "disk_image": "Existing Disk Image" }` and then generalize/simplify this code. If you don't like this, I'm still in favor of avoiding the `self.is_*` duplication and just have one source of truth with `self.sourceType`.

Reply:
I did remove the is_*. We probably  won't have much more installation modes so I 'll skip the dict for now.